tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/api/rest/v1/controllers/internal/DevUtilExternalController.java

```diff
@@ -73,7 +71,11 @@ public ResponseEntity<?> recreateData() throws IOException {
 
     final Set<String> indices =
         retryElasticsearchClient
-            .getIndexNames(tasklistProperties.getElasticsearch().getIndexPrefix() + "*")
+            .getIndexNames(
+                tasklistProperties.getElasticsearch().getIndexPrefix()
+                    + "*"
+                    + ComponentNames.TASK_LIST
+                    + "*")
```

Comment:
otherwise also non tasklist indices were deleted, e.g. breaking the login due to loss of the user index